packages/backend/src/payment-method/ilp/connector/core/test/mocks/accounting-service.ts

```diff
@@ -133,4 +135,78 @@ export class MockAccountingService implements AccountingService {
     }
     return account
   }
+
+  createDeposit(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    deposit: Deposit,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    trx?: TransactionOrKnex
+  ): Promise<void | TransferError> {
+    return Promise.resolve(undefined)
```

Comment:
We could also just throw errors for all of these to make sure they aren't used but I don't have a strong opinion

Reply:
I like that more. I updated it.